spec/rubocop/cop/rails/to_s_with_argument_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::ToSWithArgument, :config, :rails70 do
+  context 'without argument' do
+    it 'does not register an offense' do
+      expect_no_offenses(<<~RUBY)
+        to_s
+      RUBY
+    end
+  end
+
+  context 'with argument' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        to_s(:delimited)
+        ^^^^^^^^^^^^^^^^ Use `to_formatted_s(...)` instead of `to_s(...)`.
```

Comment:
Seems to show a concrete value.

```suggestion
        ^^^^^^^^^^^^^^^^ Use `to_formatted_s(:delimited)` instead of `to_s(:delimited)`.
```

Reply:
OK, this time I've decided to change it to the same format as `Rails/ToFormattedS`.

```
1.to_s(:delimited)
  ^^^^ Use `to_formatted_s` instead.
```